SillyBusinessInc/SillyBusinessGame
# Feat/#151 melee enemy modelanimations import

## Purpose of this PR:
This PR imports the MeleeEnemy model (toast_anim), and implements the animations provided (idle, idle_special, walk).
Also moves the involved files into their correct folders.

## How to test:
Place the new MeleeEnemy prefab into your scene. Make sure this scene has a navmeshsurface.
Confirm that in Chasing or Roaming states the walking animation is played, In the idle state there will be a random selection between idle and idle_special. While Attacking the idle_special animation should play.

### links:
closes #151
